app/models/form.rb

```diff
@@ -35,6 +34,7 @@ def initialize(form_path, name, sections_in_form = [], type = "lettings")
       @questions = pages.flat_map(&:questions)
       @start_date = Time.iso8601(form_definition["start_date"])
       @end_date = Time.iso8601(form_definition["end_date"])
+      @name = "#{start_date.year}_#{end_date.year}"
```

Comment:
Minor point, but I think it'd be good to have fields in the same order in both branches if possible (i.e. name first, etc.)

Reply:
Hm, I thought that, the reason this is at the bottom is because we're using the fields that get set on likes 35/36. I could move the sales name to the bottom tho

Reply:
Ah, yeah, missed that (was conflating `start_year` and `start_date.year`). I think it's okay as-is, given that 👍